tests/components/sleepiq/test_light.py

```diff
@@ -0,0 +1,127 @@
+"""The tests for SleepIQ light platform."""
```

Comment:
Thanks for adding tests!!!

Please remove the classes from the tests.  A good example of how tests should be written is `tests/components/wled`